simpeg_drivers/components/factories/directives_factory.py

```diff
@@ -110,6 +112,7 @@ def save_directives(self):
             "save_iteration_data_directive",
             "save_iteration_residual_directive",
             "save_iteration_apparent_resistivity_directive",
+            # "save_iteration_sensitivity_directive",
```

Comment:
Is this in here as a placeholder to when we hook up the option to save sensitivities?